code/go/0chain.net/smartcontract/dbs/goose/migrations/001_events_db.sql

```diff
@@ -1200,11 +1200,7 @@ CREATE TABLE public.write_markers (
                                       size bigint,
                                       "timestamp" bigint,
                                       signature text,
-                                      block_number bigint,
-                                      lookup_hash text,
-                                      name text,
-                                      content_hash text,
-                                      operation text
```

Comment:
We need to add new sql file to delete these field I think, @dabasov please comment

Reply:
@peterlimg I think that would be relevant in post-production. While in development phase it is natural to have such changes. I remember how blobber sql files were messy because of it.

Reply:
Yes, I think it should be safe to edit the exist before production. I'll approve